debian/control

```diff
@@ -0,0 +1,19 @@
+Source: txjuju
+Maintainer: Canonical Landscape Team <landscape-team@canonical.com>
+Section: python
+Priority: optional
+Build-Depends: dh-python, python-setuptools (>= 0.6b3), python-all (>= 2.6.6-3), debhelper (>= 9)
```

Comment:
I had to update the build-depends to get this to build with sbuild:

```
diff --git a/debian/control b/debian/control
index beebca6..be038c5 100644
--- a/debian/control
+++ b/debian/control
@@ -2,7 +2,16 @@ Source: txjuju
 Maintainer: Canonical Landscape Team <landscape-team@canonical.com>
 Section: python
 Priority: optional
-Build-Depends: dh-python, python-setuptools (>= 0.6b3), python-all (>= 2.6.6-3), debhelper (>= 9)
+Build-Depends: debhelper (>= 9),
+               dh-python,
+               python-all (>= 2.6.6-3),
+               python-fixtures,
+               python-setuptools (>= 0.6b3),
+               python-testtools,
+               python-twisted-core (>= 16.0.0),
+               python-twisted-names (>= 16.0.0),
+               python-twisted-web (>= 16.0.0),
+               python-yaml
 Standards-Version: 3.9.2
 Homepage: https://github.com/juju/txjuju
```

It also helps to keep this list sorted and one per line.